src/debugging/netSdk/NetSdkDebugHelper.ts

```diff
@@ -124,24 +110,46 @@ export class NetSdkDebugHelper extends NetCoreDebugHelper {
         return appOutput;
     }
 
-    private inferDotNetSdkContainerName(debugConfiguration: DockerDebugConfiguration): string {
-        const projFileUri = Uri.file(path.dirname(debugConfiguration.netCore.appProject));
-        return getDefaultContainerName(path.basename(projFileUri.fsPath), "dev");
+    private async getProjectInfo(debugConfiguration: DockerDebugConfiguration): Promise<string[]> {
```

Comment:
Consider making caching/retrieving project info a concept in the base class so that we can cache the info in either a standard .NET project or an SDK image build project.